lecture/build.gradle

```diff
@@ -0,0 +1,289 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        //jhipster-needle-gradle-buildscript-dependency - JHipster will add additional gradle build script plugins here
+    }
+}
+
+plugins {
+    id "checkstyle"
+    id "java"
+    id "maven-publish"
+    id "idea"
+    id "jacoco"
+    id "org.springframework.boot"
+    id "com.google.cloud.tools.jib"
+    id "com.github.node-gradle.node"
+    id "com.diffplug.spotless"
+    // this allows us to find outdated dependencies via ./gradlew dependencyUpdates
+    id "com.github.ben-manes.versions"
+    id "com.github.andygoossens.gradle-modernizer-plugin"
+    id "org.openapi.generator" version "5.1.1" // version is included since the plugin is not used by Artemis
+    id "io.spring.nohttp" version "0.0.8" // version is included since the plugin is not used by Artemis
+    //jhipster-needle-gradle-plugins - JHipster will add additional gradle plugins here
+}
+
+group = "de.tum.in.www1.artemis.lecture"
```

Comment:
The `user-management` microservice uses `de.tum.in.www1.artemis` as group. Probably either both should have the additional last element or neither of them should have it?